base/settings/R/pft_site_linker.R

```diff
@@ -0,0 +1,66 @@
+#'
+#' @title site.pft.linkage
+#' 
+#' @param pecan.xml.address pecan settings list.
+#' @param site.pft.links dataframe. Your look up table should have two columns of site and pft with site ids under site column and pft names under pft column.
+#'
+#' 
+#' @description This function creates the required tags inside pecan.xml to link sites with pfts given a look up table. If the required tags are already defined in the pecan xml then they will be updated.
+#' 
+#' @return NONE
```

Comment:
This says `return NONE` but the function returns `return(new.mset)`. Update documentation